tasks/input_sql_file.yml

```diff
@@ -0,0 +1,35 @@
+- name: Verify if postgresql_password is set
```

Comment:
This should be done in a `block` using a `tempfile` - you might be able to mostly copy the file from mssql - https://github.com/linux-system-roles/mssql/blob/master/tasks/input_sql_file.yml

Reply:
```suggestion
- name: Handle sql input file
  when: postgresql_input_file | d("") | length > 0
  block:
    - name: Verify if postgresql_password is set
      assert:
        that: postgresql_password is defined
        fail_msg: >-
          You must define postgresql_password variable to be able to access
          the database.

    - name: Create a tempfile for the SQL file on the host
      tempfile:
        state: file
      register: __postgresql_sql_tempfile
      changed_when: false

    - name: Copy the {{ postgresql_input_file }} file to the host
      copy:
        src: "{{ postgresql_input_file }}"
        dest: "{{ __postgresql_sql_tempfile.path }}"
        owner: postgres
        group: postgres
        mode: '0600'
      changed_when: false

    - name: Run SQL script
      become: yes
      become_user: postgres
      environment:
        PGPASSWORD: "{{ postgresql_password }}"
      command:
        cmd: psql -U postgres -f {{ __postgresql_sql_tempfile.path | quote }}
      register: __postgresql_output
      changed_when: false
  always:
    - name: Print the output of psql command for {{ postgresql_input_file }}
      debug:
        msg: "{{ __postgresql_output }}"
      when: __postgresql_debug_sql_output | d(false)

    - name: Remove sql script {{ postgresql_input_file }} from the host
      file:
        state: absent
        path: "{{ __postgresql_sql_tempfile.path }}"
```